src/Microsoft.Health.Dicom.Api.UnitTests/Controllers/ExportControllerTests.cs

```diff
@@ -34,17 +34,17 @@ public void GivenNullArguments_WhenConstructing_ThenThrowArgumentNullException()
             NullLogger<ExportController>.Instance));
 
         Assert.Throws<ArgumentNullException>(() => new ExportController(
-            new Mediator(t => null),
+            new Mediator(null),
```

Comment:
Let's make sure that the `ArgumentNullException` is not coming from `Mediator` instead of `ExportController`

Reply:
Extracted the object construction outside.